libraries/networking/src/udt/LossList.h

```diff
@@ -0,0 +1,52 @@
+//
+//  LossList.h
+//  libraries/networking/src/udt
+//
+//  Created by Clement on 7/27/15.
+//  Copyright 2015 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+#ifndef hifi_LossList_h
+#define hifi_LossList_h
+
+#include <list>
+
+#include "SequenceNumber.h"
+
+namespace udt {
+
+class ControlPacket;
+    
+class LossList {
+public:
+    LossList() {}
+    
+    void clear() { _length = 0; _lossList.clear(); }
+    
+    // Should always add at the end
```

Comment:
we should make this a clearer comment - it fails the assert if you're not actually appending - as a caller you must always add at the end